gadgets/trace_exec/program.bpf.c

```diff
@@ -106,6 +110,7 @@ int ig_execve_e(struct syscall_trace_enter *ctx)
 
 	event->timestamp_raw = bpf_ktime_get_boot_ns();
 	event->pid = tgid;
+	event->tid = pid;
```

Comment:
I know the values are correct, but can you please rename the local variables to match?

Reply:
I'm actually not sure about this one. I'd prefer to use kernel terminology on eBPF programs, otherwise it can become confusing. I'd say the only place where we should use user space terminology is on the events structure. 

@alban any comments on this?

Reply:
TIL. And I hate it 😄  But go ahead, then 🙈

Reply:
Welcome to the club of people confused by TGID / PID and TID :D 

I merged it, we can make another pass later on to see if the right naming is used in all eBPF programs.